project/config.py

```diff
@@ -7,3 +7,7 @@ class Config(object):  # pylint: disable=too-few-public-methods
     """Default Flask configuration."""
     DEBUG = True
     SECRET_KEY = os.environ.get('SECRET_KEY') or 'test-key'
+    TWIT_CONSUMER_KEY = os.environ.get('TWITTER_CONSUMER_KEY')
```

Comment:
Is there a reason why you used TWIT instead of TWITTER? Personally I'd prefer to just use TWITTER

Reply:
I believe it was because the latter would have too many characters in a line, so I shortened the name.

Reply:
Oh don't worry about that. That's why we use yapf haha.